django_rq/utils.py

```diff
@@ -89,7 +91,22 @@ def get_statistics(run_maintenance_tasks=False):
         queue_data['scheduled_jobs'] = len(scheduled_job_registry)
 
         queues.append(queue_data)
-    return {'queues': queues}
+
+        # there is only one scheduler per redis connection, so we use the connection as key
+        # to handle the possibility of a configuration with multiple redis connections and scheduled
+        # jobs in more than one of them
+        conn_key = f"{connection_kwargs['host']}:{connection_kwargs['port']}/{connection_kwargs['db']}"
+        if conn_key not in schedulers:
+            try:
+                scheduler = get_scheduler(config['name'])
+                schedulers[conn_key] ={
+                    'count': scheduler.count(),
+                    'index': index,
+                }
+            except ImproperlyConfigured:
+                pass
```

Comment:
Sorry, but do you mind moving this block of code to a separate `get_scheduler_statistics()`? I think the change shouldn't be that big and it feels a bit cleaner to have this separation of gathering queue and scheduler statistics (which is a separate library altogether).